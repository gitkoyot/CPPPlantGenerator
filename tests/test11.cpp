/*
 * test11.cpp
 *
 *  Created on: 30 wrz 2013
 *      Author: koyot
 */

#include "test11.h"

Test11::Test11()
{
  // TODO Auto-generated constructor stub

}

Test11::~Test11()
{
  // TODO Auto-generated destructor stub
}

int
Test11::function(int a, int b)
{
}
