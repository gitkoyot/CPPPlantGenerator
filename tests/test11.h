/*
 * test11.h
 *
 *  Created on: 30 wrz 2013
 *      Author: koyot
 */

#ifndef TEST11_H_
#define TEST11_H_

class Test11
{
public:
  Test11();
  virtual
  ~Test11();

  virtual int function(int a, int b);


  virtual void getVoidDefnitionInClass(int a, int b)
  {

  }
};

#endif /* TEST11_H_ */
