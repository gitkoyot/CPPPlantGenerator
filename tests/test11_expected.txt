@startuml
class Test11{
void Test11 ( )
void ~Test11 ( )
int function ( a   :   int , b   :   int)
void getVoidDefnitionInClass ( a   :   int , b   :   int)
}
@enduml

